region/wreckedship/main/Wrecked Ship Energy Tank Room.json

```diff
@@ -370,7 +370,118 @@
         "Run from the adjacent room and jump at the end of the platform.",
         "From the top right platform, jump to the left and wall jump just above the spikes and climb the left wall."
       ],
-      "devNote": "Because this requires canCarefulJump, there is no canRiskPermanentLossOfAccess as there is a path across with or without f_DefeatedPhantoon."
+      "devNote": [
+        "Because this requires canCarefulJump, there is no canRiskPermanentLossOfAccess as there is a path across with or without f_DefeatedPhantoon.",
+        "This variant of the strat includes an extra couple of tiles of runway for lenience."
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "Tricky Speedy Jump Wall Jump (Power Off)",
+      "entranceCondition": {
+        "comeInRunning": {
+          "speedBooster": true,
+          "minTiles": 3
+        }
+      },
+      "requires": [
+        "canTrickyJump",
+        "canConsecutiveWalljump",
+        {"not": "f_DefeatedPhantoon"}
+      ],
+      "note": [
+        "Run from the adjacent room and jump at the end of the platform.",
+        "From the top right platform, jump to the left and wall jump just above the spikes and climb the left wall."
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "Insane Speedy Jump Wall Jump (Power Off)",
+      "entranceCondition": {
+        "comeInRunning": {
+          "speedBooster": true,
+          "minTiles": 2
+        }
+      },
+      "requires": [
+        "canInsaneJump",
+        "canConsecutiveWalljump",
+        {"not": "f_DefeatedPhantoon"}
+      ],
+      "note": [
+        "Run from the adjacent room and jump at the end of the platform.",
+        "From the top right platform, jump to the left and wall jump just above the spikes and climb the left wall."
+      ],
+      "detailNote": [
+        "With minimal runway length (2 tiles), this has a 2-frame window for the jump.",
+        "With a last-frame jump it has a 3-frame window for wall jump,",
+        "otherwise the wall jump is frame-perfect."
+      ]
+    },
+    {
+      "link": [1, 2],
+      "name": "Speedy Jump Ceiling Mockball (Power Off)",
+      "entranceCondition": {
+        "comeInRunning": {
+          "speedBooster": true,
+          "minTiles": 23
+        }
+      },
+      "requires": [
+        "canTrickyJump",
+        "canMomentumConservingMorph",
+        {"or": [
+          "canInsaneJump",
+          {"thornHits": 5}
```

Comment:
What does 5 thorn hits mean?

Reply:
- An airball version should be added, without `canMomentumConservingMorph`, with about 3 more tiles of runway.
- A full speed jump version, with no items but speed, seems like it should be added as well.  You have to time a pose change somewhat well to make it.
- A dev note should be added to explain `thornHits` are listed for leniency.

UN farm looked approximately frame perfect on the jump.  If you rush everything you can make it back to the farm on no tanks, which is nice for retries and escaping with the item, and good that expert gets an extra etank.
I don't know how many runways have 23 tiles but not 26 for an easier jump.